File: apps/s-2-admin/src/api/credential/CredentialCreateInput.ts
import { InputJsonValue } from "../../types";
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";
import { AppModelWhereUniqueInput } from "../appModel/AppModelWhereUniqueInput";
import { DestinationCalendarCreateNestedManyWithoutCredentialsInput } from "./DestinationCalendarCreateNestedManyWithoutCredentialsInput";

export type CredentialCreateInput = {
  typeField: string;
  key: InputJsonValue;
  user?: UserWhereUniqueInput | null;
  appField?: AppModelWhereUniqueInput | null;
  destinationCalendars?: DestinationCalendarCreateNestedManyWithoutCredentialsInput;
};

File: apps/cactus-demo-dotnet/src/APIs/Credential/Dtos/CredentialCreateInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class CredentialCreateInput
{
    public int? Id { get; set; }

    public string TypeField { get; set; }

    public string Key { get; set; }

    public User? User { get; set; }

    public AppModel? AppField { get; set; }

    public List<DestinationCalendar>? DestinationCalendars { get; set; }
}
